package.json

```diff
@@ -118,5 +118,5 @@
     "packages": [
       "packages/*"
     ]
-  }
+  } 
```

Comment:
y u add white space

Reply:
Hmm our format script should've fixed this...